pybotx_widgets/checktable.py

```diff
@@ -0,0 +1,93 @@
+from dataclasses import dataclass
+from typing import Any, Dict, List, Optional
+
+from botx import Bot, Message, MessageMarkup
+
+from pybotx_widgets.resources import strings
+from pybotx_widgets.service import merge_markup, send_or_update_message
+
+
+@dataclass
+class CheckboxContent:
+    """Checkbox content."""
+
+    #: text that will be shown on the element.
+    label: str
+    #: command that will be triggered by click on the element.
+    command: str
+    #: value which will be displayed on the button associated with this checkbox.
+    checkbox_value: Any = None
+    #: list of available values.
+    mapping: Optional[Dict[Any, str]] = None
+    #: extra payload that will be stored in button and then received in new message.
+    data: Optional[Dict[str, Any]] = None
+    #: can the value be Null
+    nullable: bool = False
```

Comment:
Suggest adding some validation to ensure that checkbox_value is not None when nullable=False.
Mb pydantic model is more suitable from this point of view.